doc/source/whatsnew/v0.24.0.txt

```diff
@@ -7,6 +7,7 @@ v0.24.0
 
 New features
 ~~~~~~~~~~~~
+- :func: merge now allows  a ``DataFrame`` and a ``Series`` with a name as inputs (:issue:`21220`)
```

Comment:
this won't render, use ``:func:`merge` ``. also not clear what the change is from a user POV here

Reply:
Thanks - updated the comment as:

```:func:`merge` now directly allows merge between objects of type ``DataFrame`` and ``Series`` with a name, without need to convert the ``Series`` object into a ``DataFrame`` beforehand (:issue:`21220`)```